koalixcrm/crm/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,3013 @@
+msgstr "Telefone Para Documentos de Vendas"
+
+
+#: koalixcrm/crm/documents/sales_document.py:436
+msgid "Create PDF"
+msgstr "Criar PDF"
+
+
+#: koalixcrm/crm/documents/sales_document.py:447
+msgid "Create Project"
+msgstr "Criar Projeto"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:10
+msgid "Position Number"
+msgstr "Posição Número"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:12
+msgid "Quantity"
+msgstr "Quantidade"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:22
+#: koalixcrm/crm/product/currency_transform.py:16
+msgid "Product"
+msgstr "Produto"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:25
+#: koalixcrm/crm/product/price.py:16 koalixcrm/crm/product/product_type.py:23
+#: koalixcrm/crm/product/unit.py:28
+msgid "Unit"
+msgstr "Unidade"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:28
+msgid "Shipment on"
+msgstr "Expedição em"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:31
+msgid "Overwrite Product Price"
+msgstr "Sobrescrever o Preço do Produto"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:32
+#: koalixcrm/crm/product/price.py:27
+msgid "Price Per Unit"
+msgstr "Preço por Unidade"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:36
+msgid "Last Pricing Date"
+msgstr "Ültima Data de Precificação"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:41
+msgid "Last Calculated Price"
+msgstr "Último Preço Calculado"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:45
+msgid "Last Calculated Tax"
+msgstr "Último Imposto Calculado"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:50
+#: koalixcrm/crm/documents/sales_document_position.py:55
+msgid "Position"
+msgstr "Posição"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:56
+msgid "Positions"
+msgstr "Posições"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:64
+msgid "Position in Sales Document"
+msgstr "Posição no Documetno de Vendas"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:65
+msgid "Positions Sales Document"
+msgstr "Posições no Documento de Vendas"
+
+
+#: koalixcrm/crm/documents/sales_document_position.py:99
+msgid "Sales Document Position"
+msgstr "Posição no Documento de Vendas"
+
+
+#: koalixcrm/crm/management/commands/koalixcrm_install_defaulttemplates.py:35
+msgid "File not found:"
+msgstr "Arquivo não encontrado:"
+
+
+#: koalixcrm/crm/management/commands/koalixcrm_install_defaulttemplates.py:36
+msgid "Run collectstatic command and fix potential errors"
+msgstr "Execute o comando collectstatic e corrija possíveis erros"
+
+
+#: koalixcrm/crm/management/commands/koalixcrm_install_defaulttemplates.py:66
+msgid "John Smit, Sample Company, 8976 Smallville"
+msgstr "John Smit, Compania Exemplo, 8976 Uberlândia"
+
+
+#: koalixcrm/crm/management/commands/koalixcrm_install_defaulttemplates.py:68
+msgid ""
+      "According to your wishes the contract consists of the following positions:"
```

Comment:
GitHub shows here a syntax error, but in my opinion this is valid code. Maybe it's because of the tab at the beginning of the line. @scaphilo: Any idea? Further down in the file there is the same syntax error again.